src/game.cpp

```diff
@@ -170,6 +170,12 @@ void Game::drawBoard() {
   drawAscii();
   drawScoreBoard(std::cout);
 
+  char *ptr;
+  int numlen = snprintf(ptr,0,"%d",largestTile);
+  std::string border;
+  for(int i = 0; i < numlen+2; i++){
```

Comment:
I added an enum for the cell border, so you can set the number of spaces between the cell wall and the number. I multiply it by 2 to apply the border to both the left and right side of the number. I figured that it isn't a magic number in the same sense before, because multiplying by 2 now only saves me from having to add the enum together twice